validation/standalone-constraint-constraint.ttl

```diff
@@ -214,6 +215,142 @@
     ] ;
 .
 
+:Annotation
+    a sh:NodeShape ;
+    sh:targetObjectsOf meta:annotation ;
+    sh:property
+        [
+            sh:path rdf:type ;
+            sh:minCount 1 ;
+            sh:message "annotation needs an rdf:type" ;
+        ],
+        [
+            sh:path meta:annotationContext ;
+            sh:nodeKind sh:BlankNode ;
+        ],
+        [
+            sh:path
+                (
+                    [ sh:inversePath meta:annotation ]
+                ) ;
+            sh:class cube:MeasureDimension ;
+            sh:message "annotated dimension must be a cube:MeasureDimension" ;
+        ] ;
+.
+
+:AnnotationValue
+    a sh:NodeShape ;
+    sh:targetObjectsOf meta:annotation ;
+    sh:message "annotation must have exactly one schema:value or a combination of schema:minValue and schema:maxValue" ;
+    sh:xone
+        (
+            [
+                sh:property
+                    [
+                        sh:path schema:value ;
+                        sh:minCount 1 ;
+                        sh:maxCount 1 ;
+                    ] ;
+            ]
+            [
+                sh:property
+                    [
+                        sh:path [ sh:alternativePath ( schema:minValue schema:maxValue ) ] ;
+                        sh:minCount 1 ;
+                        sh:maxCount 2 ;
+                    ] ;
+            ]
```

Comment:
@tpluscode IIUC this shacl will allow single upperLimit w/o lowerLimit, correct? 🤔  - which is what @Kronmar-Bafu expects 👍 in order to allow logical checks if values are exceeding the limit ... and style charts accordingly.

i.e. emission limits typically are of kind *upperLimit* - emssions are expected to stay *below*